bugsnag-android-core/src/test/java/com/bugsnag/android/StacktraceSerializationTest.kt

```diff
@@ -18,10 +18,10 @@ internal class StacktraceSerializationTest {
                 "stacktrace",
 
                 // empty stacktrace element ctor
-                Stacktrace(arrayOf(), emptySet(), NoopLogger),
+                    Stacktrace.stacktraceFromJavaTrace(arrayOf(), emptySet(), NoopLogger),
```

Comment:
style nit: the indentation is inconsistent with other parameters

Reply:
For some reason Android Studio insists on this indentation, and backspacing just moves the whole thing to the previous line. I had to manually select 4 spaces and cut them to get back to the original indentation :/